src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,57 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+function grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+    return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
+end
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal([ring::MPolyRing], subspace_dimension::Int, ambient_dimension::Int)
+
+Given an ambient dimension and a subspace dimension return the ideal
+generated by the Plücker relations. If the ring is not specified return the ideal 
```

Comment:
```suggestion
Given a ring, an ambient dimension and a subspace dimension return the ideal in the given ring
generated by the Plücker relations. If the ring is not specified return the ideal 
```
Perhaps also mention what happens if the ring *is* given?